app/og/route.tsx

```diff
@@ -1,22 +1,21 @@
-import { ImageResponse } from 'next/og'
+import React from 'react';
```

Comment:
Do we know if we still can take advantage of dynamic og images when statically building the next site?